### PR TITLE
feat(Query): Added Alicloud RAM Security Preference Not Enforce MFA Login for Terraform

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
-Added Alicloud RAM Security Preference Not Enforce MFA Login for Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/alicloud/ram_security_preference_not_enforce_mfa/query.rego [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	ram_users := [ram_users | docs := input.document; docs[i].resource.alicloud_ram_user[name]; ram_users := {"id": docs[i].id, "name": name}]
+	count(ram_users) > 0
+    ram_user := ram_users[0]
+    not has_preference(input.document) 
+	#resource alicloud_ram_user
```
Comment (cxMiguelSilva) [inline assets/queries/terraform/alicloud/ram_security_preference_not_enforce_mfa/query.rego:10]: apagar comment

File: assets/queries/terraform/alicloud/ram_security_preference_not_enforce_mfa/query.rego [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+has_ram_user(doc){
+	doc[_].resource["alicloud_ram_user"]
+}
+
+CxPolicy[result] {
+	has_ram_user(input.document)
+	resource := input.document[id].resource.alicloud_ram_security_preference[name]
+    not common_lib.valid_key(resource,"enforce_mfa_for_login")
```
Comment (cxMiguelSilva) [inline assets/queries/terraform/alicloud/ram_security_preference_not_enforce_mfa/query.rego:32]: add blank line

File: assets/queries/terraform/alicloud/ram_security_preference_not_enforce_mfa/query.rego [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		"keyActualValue": "'enforce_mfa_for_login' is not defined",
+		"searchLine": common_lib.build_search_line(["resource", "alicloud_ram_security_preference", name], []),
+	}
+}
+
+CxPolicy[result] {
+	has_ram_user(input.document)
+	resource := input.document[id].resource.alicloud_ram_security_preference[name]
+    resource.enforce_mfa_for_login == false
+	result := {
```
Comment (cxMiguelSilva) [inline assets/queries/terraform/alicloud/ram_security_preference_not_enforce_mfa/query.rego:47]: add blank line

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**